src/OpenTelemetry.Instrumentation.Wcf/OpenTelemetry.Instrumentation.Wcf.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>$(NetStandardMinimumSupportedVersion);$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
+    <TargetFrameworks>net6.0;$(NetStandardMinimumSupportedVersion);$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
```

Comment:
Why we would put `net6.0` back after going through all the effort to remove `net6.0`? 🤣 Let's just add `net8.0`.

I actually have most of this done locally too just never got around to opening PR because there is one bit I'm not sure of.

If you want my project file changes here is what I did. For `net8.0` I put the packages to the safe `8.0.0`+ versions.

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
    <TargetFrameworks>net8.0;$(NetStandardMinimumSupportedVersion);$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
    <Description>OpenTelemetry instrumentation for WCF.</Description>
    <PackageTags>$(PackageTags);distributed-tracing;WCF</PackageTags>
    <MinVerTagPrefix>Instrumentation.Wcf-</MinVerTagPrefix>
  </PropertyGroup>

  <!-- Do not run Package Baseline Validation as this package has never released a stable version.
  Remove this property once we have released a stable version and add PackageValidationBaselineVersion property. -->
  <PropertyGroup>
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFramework)' == '$(NetFrameworkMinimumSupportedVersion)'">
    <Reference Include="System.Configuration" />
    <Reference Include="System.ServiceModel" />
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFramework)' == '$(NetStandardMinimumSupportedVersion)'">
    <PackageReference Include="System.ServiceModel.Primitives" Version="4.7.0" />

    <!-- System.Security.Cryptography.Xml is indirect reference. It is needed to upgrade it directly to avoid https://github.com/advisories/GHSA-rxg9-xrhp-64gj -->
    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
    <PackageReference Include="System.ServiceModel.Primitives" Version="8.0.0" />

    <!-- System.Security.Cryptography.Xml is indirect reference. It is needed to upgrade it directly to avoid https://github.com/advisories/GHSA-555c-2p6r-68mm -->
    <PackageReference Include="System.Security.Cryptography.Xml" Version="8.0.2" />
  </ItemGroup>

  <ItemGroup>
    <Compile Include="$(RepoRoot)\src\Shared\AssemblyVersionExtensions.cs" Link="Includes\AssemblyVersionExtensions.cs" />
    <Compile Include="$(RepoRoot)\src\Shared\ExceptionExtensions.cs" Link="Includes\ExceptionExtensions.cs" />
    <Compile Include="$(RepoRoot)\src\Shared\Guard.cs" Link="Includes\Guard.cs" />
    <Compile Include="$(RepoRoot)\src\Shared\PropertyFetcher.cs" Link="Includes\PropertyFetcher.cs" />
  </ItemGroup>

</Project>
```

Reply:
As discussed in private channel, I would keep support for .NET6 for one release. Just after, I will bump it to .NET8.